test/e2e/placement_selecting_resources_test.go

```diff
@@ -138,3 +142,453 @@ var _ = Describe("creating CRP and selecting resources by label", Ordered, func(
 		Eventually(finalizerRemovedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to remove controller finalizers from CRP %s", crpName)
 	})
 })
+
+var _ = Describe("validating CRP when cluster-scoped resources become selected after the updates", Ordered, func() {
```

Comment:
will we see some failed cases (like failed to appy CR)?

Reply:
will do in next PR.